.markdownlint.json

```diff
@@ -0,0 +1,38 @@
+{
+    "_docs_for_this_file": "https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md",
+    "line-length": false,
+    "first-line-h1": true,
+    "first-header-h1": {
+        "level": 1
+    },
+    "header-style": {
+        "style": "atx"
+    },
+    "ul-style": {
+        "style": "asterisk"
+    },
+    "hr-style": {
+        "style": "---"
+    },
+    "fenced-code-language": true,
+    "required-headers": {
+        "headers": [
+            "*",
+            "## Flow",
+            "## Included components",
+            "## Featured technologies",
+            "## Watch the Video",
+            "## Steps",
+            "*",
+            "## License"
+        ]
+    },
+    "spelling": false,
```

Comment:
this isn't a thing? https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md